### DISCUSSION TITLE
Support for Soketi App Manager📡

### DISCUSSION THREAD

**Discussion by thereis** [coollabsio/coolify#4678] [Service Template Requests]
Since we do have the possibility to create a new soketi service with coolify, it would be interesting to have a service to manage this application accordingly, so Soketi App Manager could be one very good option to manage a soketi instance.

[Soketi App Manager 📡](https://github.com/rahulhaque/soketi-app-manager-filament)

**Comment by MajidRaimi** [comment]
Hi there,

I know this might not be entirely relevant to this repository, but this is the only resource I could find after a lot of searching, apologies if it’s out of place.

I saw that you managed to get Soketi running perfectly with Coolify, and I’d really appreciate your help. I’m trying to set up a Soketi instance via Coolify as well, but I’m a bit stuck on how to connect it to the pusher-js SDK.

Specifically, I’m not sure which credentials (app ID, key, secret, host, etc.) I need to use when initializing the Pusher object on the client side. Could you please share how you hooked it up? Any pointers or configuration examples would be a huge help!

Thanks a lot in advance.
